hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/fees/SimpleFeesSuite.java

```diff
@@ -157,14 +160,14 @@ final Stream<DynamicTest> updateTopicComparison() {
                             .adminKeyName(ADMIN)
                             .fee(ONE_HBAR)
                             .via("create-topic-admin-txn"),
-                    validateChargedUsd("create-topic-admin-txn", ucents_to_USD(1630)),
+                    validateChargedUsd("create-topic-admin-txn", ucents_to_USD(2100)),
```

Comment:
Why cant we directly give the value here in USD? Instead of `ucents_to_USD(2100)`